apps/server/src/app/routes/auth-user.router.ts

```diff
@@ -0,0 +1,6 @@
+import { Router } from 'express'
+
+// Placeholder router if needed
+const router = Router()
```

Comment:
Let's not add this file since it's not currently used, avoid future proofing code.